src/Locations.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Locations from './Locations';
+
+import locations from '../fixtures/locations';
+
+describe('Location', () => {
+  const onClick = jest.fn();
+  it('지역을 출력합니다.', () => {
+    const { queryByText } = render((<Locations locations={locations} />));
+
+    locations.forEach((location) => {
+      expect(queryByText(location.name)).not.toBeNull();
+    });
+  });
+
+  it('onClick함수가 실행합니다.', () => {
+    const { getByText } = render((<Locations onClick={onClick} />));
```

Comment:
```suggestion
    const { getByText } = render((<Locations locations={locations} onClick={onClick} />));
```

locations를 넘겨줘야할 것 같아요.

테스트마다 반복되는 render 부분은 함수로 바깥에서 관리해도 좋을 것 같아요~

Reply:
네 수정하도록 하겠습니다.